public/app/core/components/sidemenu/BottomNavLinks.tsx

```diff
@@ -56,7 +56,7 @@ export default class BottomNavLinks extends PureComponent<Props, State> {
           </span>
         </a>
         <ul className="dropdown-menu dropdown-menu--sidemenu" role="menu">
-          {link.subTitle && (
+          {link.subTitle && false && (
```

Comment:
I guess this is to disable the random slogan temporarily. Wouldn't it be good to have a comment specifying that?

Reply:
Good point. Will do )